nms/app/theme/default.ts

```diff
@@ -216,257 +217,381 @@ export default createTheme({
+        },
       },
     },
-    MuiSelect: {
-      root: {
-        display: 'flex',
-        alignItems: 'center',
+    MuiMenu: {
+      styleOverrides: {
+        root: {
+          '& .MuiBackdrop-root': {
+            backgroundColor: 'transparent',
+          },
+        },
       },
-      outlined: {
-        '&:focus': {
-          backgroundColor: colors.primary.white,
+    },
+    MuiMenuItem: {
+      styleOverrides: {
+        root: {
+          '&.Mui-selected, &.Mui-selected&:hover': {
+            backgroundColor: alpha(colors.primary.mirage, 0.05),
+          },
+        },
+      },
+    },
+    MuiSelect: {
+      styleOverrides: {
+        select: {
+          display: 'flex',
+          alignItems: 'center',
         },
-        '&:read-only': {
-          opacity: 1,
+        outlined: {
+          '&:focus': {
+            backgroundColor: colors.primary.white,
+          },
+          '&:read-only': {
+            opacity: 1,
+          },
         },
       },
     },
     MuiSwitch: {
-      colorSecondary: {
-        '&$checked': {
-          color: colors.secondary.dodgerBlue,
-        },
-        '&$checked + .MuiSwitch-track': {
-          backgroundColor: colors.secondary.dodgerBlue,
-          opacity: 0.5,
+      styleOverrides: {
+        switchBase: {
+          '&.Mui-checked': {
+            color: colors.primary.comet,
+          },
+          '&.Mui-checked + .MuiSwitch-track': {
+            backgroundColor: colors.primary.comet,
+            opacity: 0.5,
+          },
         },
       },
     },
     MuiTabs: {
-      indicator: {
-        height: '4px',
+      styleOverrides: {
+        indicator: {
+          height: '4px',
+        },
       },
     },
     MuiTab: {
-      root: {
-        textTransform: 'none',
-        ...typography.body1,
-        padding: '12px 24px',
+      styleOverrides: {
+        root: {
+          textTransform: 'none',
+          fontFamily: '"Roboto", "Helvetica", "Arial", sans-serif',
+          fontWeight: 500,
+          fontSize: '18px',
+          lineHeight: 1.5,
+          letterSpacing: '0.5px',
+          padding: '12px 24px',
+          color: colors.primary.nobel,
+          '&.Mui-selected': {
+            color: colors.primary.white,
+          },
+          '&.Mui-disabled': {
+            color: colors.primary.nobel,
+          },
+        },
       },
     },
     MuiTableRow: {
-      root: {
-        backgroundColor: 'white',
+      styleOverrides: {
+        root: {
+          backgroundColor: colors.primary.white,
+        },
       },
     },
     MuiIconButton: {
-      root: {
-        color: colors.secondary.dodgerBlue,
+      styleOverrides: {
+        root: {
+          color: colors.primary.comet,
+        },
       },
     },
     MuiAvatar: {
-      colorDefault: {
-        backgroundColor: '#e4f0f6',
-        color: colors.secondary.dodgerBlue,
+      styleOverrides: {
+        colorDefault: {
+          backgroundColor: '#e4f0f6',
```

Comment:
Can we not use a standard color here, instead of yet another new one?
No one will notice if we use `primary.selago` here instead. And it is more uniform than too.